routes/routes.js

```diff
@@ -8,4 +8,20 @@ routes.get('/hello', (req, res) => {
 routes.get('/hello/:name', (req, res) => {
     res.send(`Hello, ${req.params.name}`);
 });
-module.exports = routes;
\ No newline at end of file
+
+routes.get('/animals', (req, res) => {
+    const species = ['elephant', 'cat', 'dog', 'snake', 'bee', 'bird'];
+    const getRandomSpecies = () => {
```

Comment:
Move this function out of here.
Also:
- function name suggests that it returns an array - it's not a case,
- create a function out of here which will return array of random species as well, no reason to put that in routes logic,
- you will have duplicates. In this case, it would be better to shuffle the array (reorganize).

Reply:
You might wanna use https://lodash.com/docs/4.17.10#shuffle, but careful - don't install the whole lodash, just this function!

Reply:
To the first point 
> - function name suggests that it returns an array - it's not a case,

[species is uncountable](http://english-simply.blogspot.com/2012/06/nouns-which-are-always-are-in-plural.html) so it may be used as single and as plural.
As Dammic mentioned you indeed can extract this function if you want.

Reply:
Ok, I deleted this funcion and used lodash/sample.